### PR TITLE
chore(filters): prepare filters for v3

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter.tsx [hunk 1]
```diff
@@ -26,10 +33,15 @@ export const SizeFilter: React.FC = () => {
     filterContext.setFilter("sizes", sizes)
   }
 
+  const tokens = useThemeConfig({
+    v2: { my: 0.5, secondaryVariant: "small" as TextVariant },
+    v3: { my: 1, secondaryVariant: "xs" as TextVariant },
+  })
```
Comment (anandaroop) [inline src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter.tsx:39]: So these are all temporary shims, and eventually for example L44 becomes just `variant="xs"` and so on, yeah?

**Review by anandaroop — APPROVED** [review]
LGTM

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter.tsx (hunk 1 above)
Comment (dzucconi) [reply src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter.tsx:39]: Exactly right. Any changes are encapsulated either in this hook or if needing to switch inside of a styled-component `getThemeConfig` and then once we transition we just inline the v3 values and that's that.

File: src/v2/Components/v2/ArtworkFilter/ArtworkFilters/InstitutionFilter.tsx [hunk 2]
```diff
@@ -7,6 +13,11 @@ export const InstitutionFilter: FC = () => {
   const { aggregations, ...filterContext } = useArtworkFilterContext()
   const items = aggregations.find(agg => agg.slice === "INSTITUTION")
 
+  const tokens = useThemeConfig({
+    v2: { my: 0.5 },
+    v3: { my: 1 },
+  })
```
Comment (damassi) [inline src/v2/Components/v2/ArtworkFilter/ArtworkFilters/InstitutionFilter.tsx:19]: Not so bad

**Pull request merged**